mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -248,45 +247,44 @@ def apply_storey_filter():
     def write_application_event(
         self,
         event: dict,
-        kind: mm_constants.WriterEventKind = mm_constants.WriterEventKind.RESULT,
-    ):
+        kind: mm_schemas.WriterEventKind = mm_schemas.WriterEventKind.RESULT,
+    ) -> None:
         """Write a single result or metric to TSDB"""
 
-        event[mm_constants.WriterEvent.END_INFER_TIME] = (
-            datetime.datetime.fromisoformat(
-                event[mm_constants.WriterEvent.END_INFER_TIME]
-            )
+        event[mm_schemas.WriterEvent.END_INFER_TIME] = datetime.datetime.fromisoformat(
+            event[mm_schemas.WriterEvent.END_INFER_TIME]
         )
+        index_cols_base = [
+            mm_schemas.WriterEvent.END_INFER_TIME,
+            mm_schemas.WriterEvent.ENDPOINT_ID,
+            mm_schemas.WriterEvent.APPLICATION_NAME,
+        ]
 
-        if kind == mm_constants.WriterEventKind.METRIC:
-            # TODO : Implement the logic for writing metrics to V3IO TSDB
-            return
+        if kind == mm_schemas.WriterEventKind.METRIC:
```

Comment:
As I wrote above, I would suggest to wrap this part in a different func as well (including the initialization of the `index_cols_base`.